src/components/content/teaser/teaser.stories.mdx

```diff
@@ -17,52 +15,61 @@ You can't pass in an image **and** illustration. If you pass in an image and ill
 ## Different Teaser Styles
 
 ### Regular
+
 <Canvas>
   <Story name="Regular">
-    <Teaser
-      title="Teaser Title"
-      linkTo='#'>
-      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac,
-      vestibulum at eros. Maecenas sed diam eget risus varius blandit sit amet non magna.
+    <Teaser title="Teaser Title" linkTo="#">
+      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi
+      leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam
+      eget risus varius blandit sit amet non magna.
     </Teaser>
   </Story>
 </Canvas>
 
 ### Without Link
+
 <Canvas>
   <Story name="Without Link">
-    <Teaser
-      title="Teaser Title">
-      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac,
-      vestibulum at eros. Maecenas sed diam eget risus varius blandit sit amet non magna.
+    <Teaser title="Teaser Title">
+      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi
+      leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam
+      eget risus varius blandit sit amet non magna.
     </Teaser>
   </Story>
 </Canvas>
 
 ### With Illustration
+
 <Canvas>
   <Story name="With Illustration">
-    <Teaser
-      title="Teaser Title"
-      linkTo='#'
-      illustration='radar_030'>
-      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac,
-      vestibulum at eros. Maecenas sed diam eget risus varius blandit sit amet non magna.
+    <Teaser title="Teaser Title" linkTo="#" illustration="radar_030">
+      Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi
+      leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam
+      eget risus varius blandit sit amet non magna.
     </Teaser>
   </Story>
 </Canvas>
 
 ### With Image
+
 <Canvas>
   <Story name="With Image">
     <Teaser
       title="Teaser Title"
-      linkTo='#'
-      image={<img style={{ width: "100%" }} src={ExampleImage} alt="" />}
+      linkTo="#"
+      image={
+        <img
+          style={{ width: '100%' }}
+          src={'https://picsum.photos/seed/sy-image/600/400'}
+          alt=""
+        />
```

Comment:
I suggest we should increase the image size (old angular.png was 2000x900) and also use the `id` parameter to [pin to a specific image](https://picsum.photos/images).
```suggestion
        <img
          style={{ width: '100%' }}
          src="https://picsum.photos/id/1031/2000/900"
          alt=""
        />
```

Reply:
You are right about the resolution, but regarding a specific image, I suggest to stick to this seed, this way we will also always have the same image.

Reply:
> I suggest to stick to this seed, this way we will also always have the same image.

When we use a specific image identifier we will get exactly the image we specify. 
Seed uses a randomized image but will cache the result afterwards.